### PR TITLE
Updated repo with new note

### PR DESCRIPTION
**Pull request by jakerella**
This adds the new note.

You can see the new note in the notes.md file, it is awesome.

All tests - in other words, none - pass!

### CODE REVIEW

**Comment by sam-kim** [comment]
What a great change!

**Comment by swainea** [comment]
Questionable pull request

**Comment by noelleknight** [comment]
Tests are hard.

**Comment by mattgrosso** [comment]
Yeah, not a good idea.

**Comment by bobutcher** [comment]
jk lol

File: notes.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 These are some notes!
 
 Here is note #1
+And note #2
```
Comment (jakerella) [inline notes.md:4]: This line looks good, but I wonder if we need a second note

Comment (rposborne) [inline notes.md:4]: Clearly need more notes

**Comment by HaWyanHa** [comment]
# Kick Ass Font

**Comment by captainbarbosa** [comment]
💩💁🏻🦀🎃🌚⛄️🔥🍟🚨

**Comment by rposborne** [comment]
@jakerella Are you sure?

**Comment by bobutcher** [comment]
It's lit 🔥

**Comment by rposborne** [comment]
- [ ] I AM CHECkBOX

**Comment by HaWyanHa** [comment]
something about hitler

**Comment by captainbarbosa** [comment]
keep it 💯

**Pull request merged**